src/theme/card.m.css

```diff
@@ -0,0 +1,5 @@
+.root {
+}
+
+.actions {
```

Comment:
@nicknisi you're going to need to give this all the classnames we put into the ccm card in order to make it themable / extensible